scripts/environment/pull.sh

```diff
@@ -6,6 +6,7 @@ EXTERNAL_IMAGES=(
 )
 
 PUBLIC_IMAGES=(
+  hmctspublic.azurecr.io/am/role-assignment-service:latest
```

Comment:
Oops, does that mean we were not pulling the latest changes?

Reply:
Well we were pulling the image at the point when the helmchart was being deployed.
This adds it to the pre-pull script.